package.json

```diff
@@ -65,18 +69,18 @@
     "eslint": "^8.38.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-header": "^3.1.1",
-    "eslint-plugin-jsdoc": "^41.1.1",
+    "eslint-plugin-jsdoc": "^50.7.1",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-tsdoc": "^0.2.17",
     "fs-extra": "^9.1.0",
-    "handlebars-helpers": "^0.10.0",
+    "handlebars": "4.7.7",
     "mocha": "^8.4.0",
     "nock": "^13.2.2",
     "nyc": "^15.1.0",
     "prettier": "^2.5.1",
     "sinon": "^9.2.4",
-    "snyk": "^1.836.0",
+    "snyk": "^1.996.0",
     "sort-package-json": "^1.53.1",
+    "source-map-support": "^0.5.21",
```

Comment:
This is required to be able to run tests with npm. Not sure why this wasn't required when I was running the tests through yarn.